languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/de-CH/grammar.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="../../../../../../../../../../languagetool-core/src/main/resources/org/languagetool/rules/print.xsl" title="Pretty print"?>
+<?xml-stylesheet type="text/css" href="../../../../../../../../../../languagetool-core/src/main/resources/org/languagetool/rules/rules.css" title="Easy editing stylesheet"?>
+
+<!--
+  ~ LanguageTool, a natural language style checker
+  ~ Copyright (C) 2021 Stefan Viol (https://stevio.de)
+  ~
+  ~ This library is free software; you can redistribute it and/or
+  ~ modify it under the terms of the GNU Lesser General Public
+  ~ License as published by the Free Software Foundation; either
+  ~ version 2.1 of the License, or (at your option) any later version.
+  ~
+  ~ This library is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this library; if not, write to the Free Software
+  ~ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+  ~ USA
+  -->
+
+<!DOCTYPE rules [
+        <!ENTITY anfauf "„|&quot;|»|›|‚|'">
+        <!ENTITY anfzu "‘|“|‹|«|&quot;|'">
+        ]>
+
+<!--suppress ProblematicWhitespace -->
+<rules lang="de-CH" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="../../../../../../../../../../languagetool-core/src/main/resources/org/languagetool/rules/rules.xsd">
+
+    <!-- ====================================================================== -->
+    <!-- Tippfehler -->
+    <!-- ====================================================================== -->
+    <category id="TYPOS" name="Mögliche Tippfehler">
+        <rulegroup id="TYPOGRAPHIC_QUOTES_de_CH" name="Anführungszeichen: &quot;xyz&quot; -> «xyz»">
+            <rule>
+                <!-- https://forum.languagetool.org/t/typografische-anfuhrungszeichen-fur-de-ch-mit-giullemets/7526/3 -->
+                <pattern>
+                    <token>"</token>
+                    <token spacebefore="no"/>
```

Comment:
This will only match a single word (yes, just like the German rule). Maybe we can find a better solution for phrases? Or will it automatically work because there's some other rule for that?